docker/trustlines/docker-compose.yml

```diff
@@ -32,12 +32,10 @@ services:
     restart: unless-stopped
 
   relay:
+    image: trustlines/relay
```

Comment:
no tag needed here? does it default to latest?

Reply:
yes and yes. see commit message for my reasoning.